www/templates/intro/consent.html

```diff
@@ -1,11 +1,15 @@
 <ion-content>
 <div class="consent-title">
-<center><h4>CCI: Assessing the Travel Demand and Co-Benefit Impacts of Affordable Transit-Oriented Development</h4></center>
+<center><h4>ELCITA and CiSTUP: Electronics City Travel Behavior Study</h4></center>
 </div>
 <div class="consent-text">
 <h3>Introduction and Purpose</h3>
-I would like to invite you to take part in my research study about travel demands of affordable transit-oriented development. The principal investigator of this project is Professor Karen Chapple. This study evaluates the benefits of living in affordable housing near transit.
-
+	Thank you for installing e-mission. e-mission will collect data on your location and speed to make inferences about your trip such as mode choice. Please keep e-mission on your phone for at least two days. At the end of two days, you are requested to pull up your trips and answer questions to confirm inferences made by e-mission.
+	
+	If you are joining a study, please return to your study instructions and complete the second step before proceeding.
```

Comment:
You definitely don't need this. They should have already clicked the link to get here.